src/generator.rs

```diff
@@ -44,9 +44,21 @@ impl<'a> Generator<'a> {
         handler: impl ContentRetriever + 'static,
     ) -> Result<Generator<'a>, Error> {
         let handler = Box::new(handler);
-        let builder = EpubBuilder::new(ZipLibrary::new()?)?;
         let config = Config::from_render_context(ctx)?;
 
+        // Проверка epub_version перед созданием builder
```

Comment:
```suggestion
```

Some readers won't be able to read the comment, so it's better off without it.

Reply:
done. removed